src/index.rs

```diff
@@ -185,23 +185,17 @@ impl Index {
 
         database
       }
-      Err(redb::Error::Io(error)) if error.kind() == io::ErrorKind::NotFound => {
-        let database = unsafe {
-          Database::builder()
-            .set_write_strategy(if cfg!(test) {
-              WriteStrategy::Checksum
-            } else {
-              WriteStrategy::TwoPhase
-            })
-            .create_mmapped(&path)?
-        };
-        let tx = database.begin_write()?;
+      Err(_) => {
+        let database = Database::builder()
+          .set_cache_size(17179869184)
+          .create(&path)?;
+
+        let mut tx = database.begin_write()?;
 
-        #[cfg(test)]
-        let tx = {
-          let mut tx = tx;
-          tx.set_durability(redb::Durability::None);
-          tx
+        if cfg!(test) {
+          tx.set_durability(redb::Durability::Eventual);
+        } else {
+          tx.set_durability(redb::Durability::Immediate);
```

Comment:
Before we had None for tests, now we have immediate - was that intended?

Reply:
None for tests and Immediate for everything else